docs/environment.yml

```diff
@@ -0,0 +1,6 @@
+# Conda environment definition to build documentation on RTD
```

Comment:
How about something like `rtd_conda.yml` to make it more explicit about the purpose?